.github/workflows/lint.yml

```
@@ -32,65 +56,99 @@ env:
   CARGO_TERM_COLOR: always
 
   # Python: Tell ruff that we're in GitHub Actions, so that it emits annotations.
-  RUFF_FORMAT: github
+  RUFF_FORMAT: github 
```

Trailing space snuck in 🙂 
```suggestion
  RUFF_FORMAT: github
```